site/profile/manifests/ccs/postfix.pp

```diff
@@ -8,6 +8,12 @@
     ensure  => 'present',
     content => $auth,
   }
+  exec { 'sed -i \'/^::1/d\' /etc/hosts':
```

Comment:
Puppet has a native {{host}} type for managing {{/etc/hosts}} entries.  See: https://puppet.com/docs/puppet/5.5/types/host.html  We are also already using a {{hosts}} (plural) module for managing {{/etc/hosts}} entries via hiera.  See https://github.com/lsst-it/lsst-itconf/blob/master/hieradata/node/atsdaq-mgmt.cp.lsst.org.yaml#L41-L43 for an example of doing it via hiera.  {{exec}} is generally a last resort operation.

Reply:
I don't know why that change was on this branch, maybe just expediency.
A simpler way to get the desired effect (which is "make postfix work") is perhaps just to add to cluster/comcam-ccs.yaml
`postfix::inet_protocols: "ipv4"`

Reply:
I do not know how that change got on to this branch, perhaps happened during rebasing confusion. @glennmorris Can you make your suggested change?

Reply:
It's not my change. I just restored it after it got clobbered. (By the way, there may have been other clobbered changes that we are unaware of.) It came from here: https://jira.lsstcorp.org/browse/IT-2816
I could force push (...) a new version of this branch without it?

Reply:
In the interest of moving things along, I created
https://github.com/lsst-it/lsst-itconf/pull/317
which is the same as this one, minus the /etc/hosts change.
So maybe you want to apply that one and close this one...